packages/fxa-content-server/app/scripts/templates/pair/unsupported.mustache

```diff
@@ -25,11 +22,35 @@
       </div>
     {{/isDesktopNonFirefox}}
 
-    {{^isDesktopNonFirefox}}
-      <div class="bg-image-pair-fail"></div>
+    {{#isMobile}}
+      {{#isSystemCameraUrl}}
+        <div class="bg-image-pair-fail"></div>
+          <h2 id="pair-unsupported-header" class="card-header" tabindex="-1">{{#t}}Pair using an app{{/t}}</h1>
+        <p class="mt-4">{{#t}}Did you use the system camera? You must pair from within a Firefox app.{{/t}}</p>
+      {{/isSystemCameraUrl}}
+      {{^isSystemCameraUrl}}
+        {{^isFirefox}}
+          <div class="flex bg-orange-50 px-4 py-3 rounded-lg mb-8">
+            <div class="bg-icon-warning bg-contain bg-center bg-no-repeat w-10" aria-label="{{#t}}Attention:{{/t}}" role="img"></div>
+            <p class="ps-3 text-sm">{{#t}}Oops! It looks like you’re not using Firefox.{{/t}} <a href="{{{ escapedMobileDownloadLink }}}" target="_blank" class="link-blue" glean-data-id="cad_redirect_mobile_download">{{#t}}Download Firefox now{{/t}}</a></p>
```

Comment:
I _think_ I'm probably supposed to use `unsafeTranslate` and embed the link? But these also felt okay separate.